src/app/card/info-status-card/examples/info-status-card-example.component.less

```diff
@@ -0,0 +1,24 @@
+@bottomMargin: 12px;
+.pfng-card-info-status {
+  @supports not (word-break: break-word) {
+    word-break: break-all;
+  }
+  @supports (word-break: break-word) {
+    word-break: break-word;
+  }
+}
+.cards-pf {
+  padding-bottom: 20px;
+}
+.cards-pf .row-cards-pf {
+  margin-bottom: @bottomMargin;
```

Comment:
same here with View Encapsulation

Reply:
Since this is a single page app, it only effects this particular card example, but we should use best practices.

We can add a level of specificity by applying the margins/padding to restrictive-width-example-container and fluid-width-example-container, for example.